src/Honeybee_Generate EP Output.py

```diff
@@ -27,7 +27,7 @@
 
```

Comment:
@chriswmackey Chris did you fix this component so that it works with my Earthtube component? I've realised now that you should ignore this file don't merge it into your master as this is a version I was playing with but it doesnt work